_posts/2019-09-30-team-working-agreements.md

```diff
@@ -0,0 +1,261 @@
+      form a team where each member felt empowered and excited to take ownership of our work - encouraged me to 
+      research and share the process for coming up with a team working agreement. At first I was nervous that some 
+      team members, especially those who I perceived to be more interested in technical challenges than on team-building, 
+      would not be interested in this and find that it was a waste of time. I was pleasantly surprised that not only did 
+      everyone on the team participate, but that those team members ended up being some of the more enthusiastic 
+      proponents for following-up on the team working agreement after our initial brainstorm.
+    </p>
+  </div>
+</aside>
+
+Since being pioneered by Adam's team, team working agreements have spread to most of our engineering teams. They've
+been a learning opportunity for all of us, and a chance for our team members to share experiences and practices
+with each other.
+
+## What's a team working agreement?
+
+### Steps
+
+Describe it in a few sentences:
+
+1. Teach team about team working agreements
+2. Conduct a brainstorming meeting
+
+- https://github.com/artsy/README/blob/master/playbooks/agile/team-working-agreement.md#procedure
+
+3. Distill and translate brainstorming ideas into slogan-like habits (ADAM: find themes, combine similar ones,
+   condense ideas >>> habits. habits are an amalgamation of ideas from brainstorming that fit together)
+4. Vote on the top 5 habits (ADAM: don't limit to 5. mention the output is a set of the most important habits that
+   the team can agree on)
+
+(ADAM: what should come out of each step, and what is the way you get there?) (ADAM: add a step to revisit often)
+
+## Why establish a team working agreement?
+
+### The brainstorm itself is incredibly valuable.
+
+(ADAM: call out that all team members - P, D, D, and E - should attend.)
+
+Dedicated time to talk about values, habits, and processes is incredibly helpful to a team, yet rarely scheduled.
+The brainstorm for a team working agreement can fill this void.
+
+During the team agreement brainstorm the team looks at _how_ they work, instead of looking at _what_ work they're
+doing. It's similar to what you might get out of a retro but at a higher level. It offers time to discuss "soft"
+skills that you don't often discuss, in a context that you don't usually discuss them - with other individual
+contributors, and with your closest collaborators. (ADAM: can this paragraph be reorganized to contrast the
+proactive nature of TWAs to the retroactive nature of retros?)
+
+<aside class="dialogue">
+  <div class="answer">
+    <img src="/images/dialogue/steve.jpg">
+    <p class="intro">Steve Hicks says...</p>
+    <p>
+      When joining a new team, there's often a period of "feeling it out" - where I'm afraid to verbalize an
+      opinion because I'm not sure how the team feels about it. The team working agreement process has definitely
+      helped lessen this feeling.
```

Comment:
Sounds like forming/storming/norming/performing: https://en.wikipedia.org/wiki/Tuckman%27s_stages_of_group_development

Reply:
We talked about making a reference to this but when I wrote this part I didn't find a place for it. Would it be weird to link to that article from here?

Reply:
Found a place to describe/link this in 263465cfe923584ec748a168258848116e99eb83